src/clients/SpokePoolClient.ts

```diff
@@ -482,7 +457,7 @@ export class SpokePoolClient extends BaseAbstractClient {
    * @returns The deposit ID.
    */
   public async _getDepositIdAtBlock(blockTag: number): Promise<number> {
```

Comment:
Does this function stil need to be a sync?

Reply:
The function itself returns a promise. But no, it doesn't need the `async` modifier, which won't change the execution logic.  

I'd like to handle this in a secondary change that applies the same relayer eslint file to the SDK. To do this, it would be an easier lift to do so after this & the other SDK change is merged.